test/common-test.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * File: common-test.js
+ * Unit test helper module
+ */
+
+var _ = require('underscore');
+
+var chai = require('chai');
+var should = chai.should();
+var expect = chai.expect;
+
+var exec = require('child_process').exec;
+var fs = require('fs');
+var os = require('os');
+
+module.exports = {
+
+    cmdFileNonexistent: function(cmd, file, done) {
+        exec(cmd, function (error, stdout, stderr) {
+             error.code.should.equal(1);
+             stderr.should.contain("File " + file + " does not exist!");
+             done();
+        });
+    },
+
+    cmdFileUnreadable: function(cmd, file, done) {
```

Comment:
Oh, I get it now.  How about:
* Name it verifyCmdOnNewUnreadableFile?  You might want to rename cmdFileNonexistent similarly.
* Rename 'file' parameter to 'tmpFile', to better suggest that it is temporary.
* Add a comment below: // Temporarily create an unreadable tmpFile